tests/index.test.js

```diff
@@ -26,8 +31,14 @@ icons.forEach((icon) => {
     expect(subject.source).toEqual(icon.source);
   });
 
-  test(`${icon.title} has an "svg" value`, () => {
+  test(`${icon.title} has a valid "svg" value`, () => {
     expect(typeof subject.svg).toBe('string');
+    const svgFileContents = fs
+      .readFileSync(svgPath, 'utf8')
+      .replace(/\r?\n/, '');
```

Comment:
Not necessarily in scope for this Pull Request, but given that we also do this transformation here:

https://github.com/simple-icons/simple-icons/blob/3fdcb11536ef0356f3f84594fbee835531bcd05b/scripts/build/package.js#L90

and now in both test suites, it might make sense to extract the functionality into a helper function.

Reply:
I think makes more sense to unify newlines in SVGs contents, but yes in another PR.

Reply:
Unless the unification is _no newlines_ (see also  #1682) you'll need a utility function like this (at least[^1]) to be able to build the package correctly.

[^1]: In case of all `.svg`s having a newline, I guess you can test against `${subject.svg}\n`, but that results in duplication between the two test files.